pkg/operator/ceph/object/admin.go

```diff
@@ -252,48 +266,10 @@ func runAdminCommand(c *Context, expectJSON bool, args ...string) (string, error
 		args = append(args, options...)
 	}
 
-	// work around FIFO file I/O issue when radosgw-admin is not compatible between version
-	// installed in Rook operator and RGW version in Ceph cluster (#7573)
 	result, err := RunAdminCommandNoMultisite(c, expectJSON, args...)
-	if err != nil && isFifoFileIOError(err) {
-		logger.Debugf("retrying 'radosgw-admin' command with OMAP backend to work around FIFO file I/O issue. %v", result)
```

Comment:
The FIFO file IO error was a special case of the forward compatibility problem, which we are solving more generically now by trying to use command proxying, so it seemed good to clean this up.